packaging/scripts/postinstall.sh

```diff
@@ -6,6 +6,67 @@ ETC_SERVICE="/etc/systemd/system/evcc.service"
 USR_LOCAL_BIN="/usr/local/bin/evcc"
 RESTART_FLAG_FILE="/tmp/.restartEvccOnUpgrade"
 
+
+# Call /usr/bin/evcc checkconfig and capture the output
+# if exit code is 0, then remove /tmp/oldevcc
+# if exit code is not 0, then fail installation with error message and copy /tmp/oldevcc back to /etc/evcc
+# if /tmp/oldevcc does not exist, then do nothing
+failInstallation=0
+INTERACTIVE=0
+# is shell script interactive?
+if [ -t 0 ]; then
+  INTERACTIVE=1
+else
+  INTERACTIVE=0
+fi
+
+if [ -f /tmp/oldevcc ] && [ $INTERACTIVE -eq 1 ]; then
+    checkConfigOutput=$(/usr/bin/evcc checkconfig 2>&1 || true)
```

Comment:
What's the point of the OR with `true` here?

Reply:
Thank you for your feedback. The modification to this line is intended to prevent the package manager from halting the script execution if the subcommand returns a non-zero exit code. By appending '|| true'. 

I have run some tests on the behaviour of the package manager that we can discuss (as soon as I have more time). This may result in a few more changes.

Reply:
OK interesting - especially the part about the subcommand / subshell RC influencing postinstall.sh. I would fail to see how postinstall.sh can even know about any RC from the variable substitution as it does not capture it. But I'll take your word for it.
I'll mark this as resolved, feel free to ping me again on this subject once you had the time to fiddle around more, and want to bounce around ideas.